### ISSUE TITLE
api.Element.requestPointerLock - Not supported on Chrome for Android

### ISSUE THREAD

**Issue by alesandroortiz** [mdn/browser-compat-data#19829]
### What type of issue is this?

Incorrect support data (example: Chrome says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Chrome for Android does not support the Pointer Lock API on Android devices. This probably also affects other Chromium-based browsers unless they are patched to enable this feature.

### What browsers does this problem apply to, if applicable?

Chromium (Chrome, Edge 79+, Opera, Samsung Internet)

### What did you expect to see?

Expected: If using touch device or mouse, pointer lock overlay appears warning that pointer lock has been obtained by page. If using mouse, cursor is not shown on screen. In all cases, pointer lock is successfully obtained by page.

Observed: Nothing occurs visually and mouse/touch behavior does not change

### Did you test this? If so, how?

Tested on Chrome for Android 112.0.5615.135 Stable on Android 12, using external mouse and touch screen. Neither worked with several Pointer Lock API demos, including https://mdn.github.io/dom-examples/pointer-lock/ . The Pointer Lock API does not throw an exception, but also does not lock the pointer (seems like essentially a no-op).

When unable to obtain pointer lock on Android, I think `document.pointerLockElement` is not set and `pointerlockchange` events are not emitted, but haven't tested this in isolation.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

You can test on https://mdn.github.io/dom-examples/pointer-lock/ using either touch screen or external mouse.

See this feature crbug: https://bugs.chromium.org/p/chromium/issues/detail?id=153419

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/Pointer_Lock_API

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.Element.requestPointerLock`
* Report started: 2023-05-11T16:13:31.870Z

</details>

**Comment by caugner** [comment]
@JeffreyBlanz Is this still an issue? If so, we should replace the corresponding `"mirror"` statements for Chrome Android with `{ "version_added": false }`.

**Comment by JeffreyBlanz** [comment]
@caugner Yes, there is still no pointer lock API support for Android last i checked. There is also no support on iOS/iPadOS.

This is the only thing preventing keyboard and mouse cloud gaming on mobile. I wish it was supported.

**Comment by caugner** [comment]
@JeffreyBlanz Thanks for confirming, would you like to open a PR updating the `chrome_android` statements from `"mirror"` to `false`? 🙏 (You should be able to find all features by searching this repo for [`"web-features:pointer-lock"`](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data%20%2F%22web-features%3Apointer-lock%22%2F&type=code).)

**Comment by JeffreyBlanz** [comment]
@caugner Unfortunately I do not know how to write the actual code to change the status. All I could do to help would be to create a new thread that someone else would have to contribute code to.

**Comment by caugner** [comment]
No worries, I opened https://github.com/mdn/browser-compat-data/pull/26738 with the changes.
